.changeset/cyan-dolphins-change.md

```diff
@@ -0,0 +1,29 @@
+---
+'xstate': minor
+---
+
+Tags can now be added to state node configs under the `.tags` property:
+
+```js
+const machine = createMachine({
+  initial: 'green',
+  states: {
+    green: {
+      tags: 'go' // single tag
+    },
+    yellow: {
+      tags: 'go'
+    },
+    red: {
+      tags: ['stop', 'other'] // multiple tags
+    }
+  }
+});
+```
+
+You can query whether a state has a tag via `state.hasTag(tag)`:
```

Comment:
Should this also mention `state.tags` or is that not something you want advertised?

Reply:
Not right now, I think that'll be an implementation detail for now.